vllm/model_executor/layers/mamba/mamba_mixer.py

```diff
@@ -185,126 +221,143 @@ def forward_cuda(self,
                 self_kv_cache = self.kv_cache[forward_context.virtual_engine]
                 conv_state = self_kv_cache[0].transpose(-1, -2)
                 ssm_state = self_kv_cache[1]
-                has_initial_state = mamba1_metadata.has_initial_states
-                context_lens_tensor = mamba1_metadata.context_lens_tensor
+                has_initial_states_p = mamba1_metadata.has_initial_states
```

Comment:
Just like mamba2, can mamba1_metadata.has_initial_states be a tensor with shape `(num_prefills,)`? `has_initial_states_p` is a bit confusing now because it is with shape `(num_decode+num_prefill,)`